demisto_sdk/commands/test_content/test_modeling_rule/tests/test_modeling_rule_test.py

```diff
@@ -1300,3 +1511,194 @@ def test_day_suffix(day, suffix):
     )
 
     assert day_suffix(day) == suffix
+
+
+class TestValidateSchemaAlignedWithTestData:
+    @pytest.mark.parametrize(
+        "event_data, schema_file",
```

Comment:
Please add a test that catches the case where we have the same field in 2 test data events with different types.